com.unity.cinemachine/Tests/Runtime/MockControllers/InputController1AllAxis.cs

```diff
@@ -0,0 +1,21 @@
+using System;
```

Comment:
I think the MockController tests belong in another PR

Reply:
Ok will remove.